deployment/v3/utils/prop_migrator/README.md

```diff
@@ -141,6 +164,16 @@ This comment is used when the properties which are present in the latest propert
 
 * Remember that careful review and validation of the CSV files and the resulting manual updates are essential to ensure the correctness and consistency of the  configuration. Always keep backups of the original property files and use version control systems to track changes during the update process.
 
+**Instructions to update properties using `manual-configuration.csv`**:
+
+The `manual-configuration.csv` file contains instructions for updating various property files. Each row in the CSV file corresponds to a configuration update. The columns in the CSV file are as follows:
+
+- **Update**: Specify whether to update with the 'Old Value' or the 'Latest Value'. This column should contain either 'old_value' or 'latest_value'.
```

Comment:
1. Lets rename this to manual update value
2. We should ask the explict value that needs to be updated, not just old or new value. In some case they might want to give a different value.